### PR TITLE
Remove imports of fmt and strings packages, to reduce binary size

### PR DESCRIPTION
**Pull request by pdf**
Helps #144 

Looks like `strconv` is imported by something fairly standard.  I didn't try too hard to track it down, I just removed `fmt` and `strings`.

### CODE REVIEW

File: example/todomvc/store/storeutil/storeutil.go [hunk 1]
```diff
@@ -24,7 +22,7 @@ func (r *ListenerRegistry) Add(key interface{}, listener func()) {
 		key = new(int)
 	}
 	if _, ok := r.listeners[key]; ok {
-		panic(fmt.Sprintf("listener with key already exists: %v", key))
+		panic("duplicate listener key")
```
Comment (slimsag) [inline example/todomvc/store/storeutil/storeutil.go]: should still include key here, otherwise the panic message is substantially less useful.

**Comment by slimsag** [comment]
One comment I'd like to see addressed before merging.

I've also edited PR description to remove:

> Fixes #144

Because I'd still like to figure out if removing strconv is feasible. I'll rename the issue to be just for strconv once merged.

File: example/todomvc/store/storeutil/storeutil.go (hunk 1 above)
Comment (pdf) [reply example/todomvc/store/storeutil/storeutil.go]: key is an interface, could be anything (struct, pointer, whatever).

Comment (pdf) [reply example/todomvc/store/storeutil/storeutil.go]: Also, this code is in the examples, and will probably go away at some point anyway, which is why I didn't worry too much about it.

**Pull request merged**